proven-member/disclosure-lib/src/main/java/gov/pnnl/proven/cluster/lib/disclosure/exchange/DisclosureEntryType.java

```diff
@@ -136,7 +136,7 @@ public static DisclosureEntryType getEntryType(String entry) throws UnsupportedD
 	 * 
 	 * @return the {@code DisclosureEntryType}
 	 */
-	public static DisclosureMessage getDisclosureMessage(String entry) throws UnsupportedDisclosureEntryType {
+	public static DisclosureMessage getDisclosureMessage(String entry) throws UnsupportedDisclosureEntryType, JsonParsingException, Exception {
```

Comment:
The following method getJsonEntry() should also have the additional exceptions added to the throws clause as well.  

I think CsvParsingException should also be added to the throws clauses.